aksel.nav.no/website/app/dev/produktbloggen/_ui/Produktbloggen.module.css

```
@@ -1,9 +1,132 @@
+.overviewTitle {
+  font-size: 48px;
+  color: var(--ax-text-brand-blue);
+  text-align: center;
+}
+
+.overviewSubtitle {
+  & p {
+    color: var(--ax-text-brand-blue-subtle);
+    text-align: center;
+    max-width: 670px;
+    font-size: var(--ax-font-size-heading-small);
+  }
+}
+
+.latestBloggPosts {
+  display: grid;
+  gap: var(--ax-space-48);
+  margin-inline: auto;
+  margin-block: var(--ax-space-40);
```

Should be 48 based on Figma 💄 
```suggestion
  margin-block: var(--ax-space-48);
```